kedro_vertexai/generator.py

```diff
@@ -95,12 +95,16 @@ def _generate_hosts_file(self):
         )
 
     def _create_mlflow_op(self, image, tracking_token) -> dsl.ContainerOp:
+
+        should_add_params = len(self.context.params) > 0
         mlflow_command = " ".join(
             [
                 self._generate_hosts_file(),
                 "mkdir --parents",
                 "`dirname {{$.outputs.parameters['output'].output_file}}`",
                 "&&",
+                self._generate_params_command(should_add_params),
+                "&&",
```

Comment:
It will break if `_generate_paramms_command` will return "", we will have && && in bash